agents/pxe_images/ubuntu/preseed.cfg

```diff
@@ -0,0 +1,489 @@
+#### Contents of the preconfiguration file (for stretch)
```

Comment:
where does this file come from and what license is associated with it? Are we allowed to incorporate this into our repository?

Reply:
[example preeseed from help.ubuntu.com](https://help.ubuntu.com/lts/installation-guide/example-preseed.txt)
Or we don't insert the file into the repository, but only the link above in the documentation and our command at the end of the file.